### ISSUE TITLE
Mod maker

### ISSUE THREAD

**Issue by RovarPlaysYT** [studio-minus/ppg-bugs-and-requests#912]
**Feature description**
Option in menu, with this feature you can make mods without coding.

**Feature purpose**
I request this feature, because half of players don't know how to code

**Additional context**
![1](https://user-images.githubusercontent.com/100331909/159122393-3f25a8a9-56c6-418e-9f28-7bef9c8747a2.png)

![listmod](https://user-images.githubusercontent.com/100331909/159122401-3742e06b-7c19-41f4-8544-40fbddad29bd.png)

![listobject](https://user-images.githubusercontent.com/100331909/159122410-a6a8a284-5a5c-481b-bf64-f7aef3eda456.png)

**Comment by slaterrr** [comment]
**how about dont be lazy and learn how to code**

**Comment by UniDuki** [comment]
No. I would rather not have the workshop spammed with among us mods made by 7 year olds

**Comment by DindinYT37** [comment]
Likely won't be added, if I'm bored enough, maybe I'll make a website to make basic mods with.

**Comment by vda8** [comment]
Learn to code then
